db/models.py

```diff
@@ -1 +1,10 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, primary_key=True)
```

Comment:
why did you make it a `primary_key`?

Reply:
I think that it's unnecessary to store a few genres with the same name, that's why name is primary key.